clis/teliod/src/command_listener.rs

```diff
@@ -105,13 +99,10 @@ mod tests {
 
     const TEST_SOCKET_PATH: &str = "test_socket";
 
-    // Create a "random" socket path for the test, since tests run in parallel they can deadlock
+    // Create a random socket path for the test, since tests run in parallel they can deadlock
     fn make_socket_path() -> String {
-        let nanos = SystemTime::now()
-            .duration_since(UNIX_EPOCH)
-            .unwrap()
-            .subsec_nanos();
-        format!("{}_{}", TEST_SOCKET_PATH, nanos)
+        let mut rng = rand::thread_rng();
```

Comment:
got a collision ?
Maybe it makes more secnce to have some lazy_static! BUSY_LIST ?

Reply:
I was using time nanos as "random" value to avoid additional dependencies, but they were not precise enough, 1/10 times having two test starting almost instantaneously. `rand` is simple enough to get this sorted for now.